.github/workflows/main.yml

```diff
@@ -268,7 +265,6 @@ jobs:
         echo ------------ py3 wheel - pip
         pip3 install wheel
         echo ------------ slither analyzer install
-        # pip3 install slither-analyzer
         pip3 install slither-analyzer==0.6.8
```

Comment:
It's a too old version. Use slither version from `proxy/scripts/requirements.txt`

Reply:
Will do it. Thanks.